Tests/MapboxNavigationTests/InstructionsBannerViewSnapshotTests.swift

```diff
@@ -349,14 +349,19 @@ class InstructionsBannerViewSnapshotTests: InstructionBannerTest {
             override func apply() {
                 super.apply()
                 
-                PrimaryLabel.appearance(whenContainedInInstancesOf: [InstructionsBannerView.self]).normalTextColor = UIColor.green
-                SecondaryLabel.appearance(whenContainedInInstancesOf: [InstructionsBannerView.self]).normalTextColor = UIColor.red
+                let traitCollection = UITraitCollection(userInterfaceIdiom: .phone)
+                PrimaryLabel.appearance(for: traitCollection,
```

Comment:
We could remove the added `whenContainedInInstancesOf: [InstructionsBannerView.self]` in all the custom style apply method, because we don't have this priority in the default `DayStyle`